src/nvim/api/vim.c

```diff
@@ -545,6 +545,27 @@ Object nvim_exec_lua(String code, Array args, Error *err)
   return nlua_exec(code, args, err);
 }
 
+/// Notify the user with a message
+///
+/// Relays the call to vim.notify . By default forwards your message in the
+/// echo area but can be overriden to trigger desktop notifications.
+///
+/// @param msg        Message to display to the user
+/// @param log_level  The log level
+/// @param opts       Reserved for future use.
+/// @param[out] err   Details of an error encountered while parsing
```

Comment:
typically all code would already been parsed at this point. just the typical "Error details, if any" is enough.

Reply:
done